spawnpoint.nats.js

```diff
@@ -54,6 +54,17 @@ module.exports = require('spawnpoint').registerPlugin({
 				config.connection.name += ` node@${process.version}`;
 			}
 
+			// Setup an authenticator if available
+			if(config.connection.auth) {
+				// We want to read in the creds file given to us
+				const authCredsFile = path.join(app.cwd, config.connection.auth.creds_file);
+				delete config.connection.auth.creds_file;
```

Comment:
Not sure if this is redundant as I delete the entire object on line 65 but included just in-case.